packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -192,6 +197,20 @@ async function _getDashboardStats(app) {
     dashboard.missingPatches = missingPatches
   }
 
+  const backupRepositorySize = Object.values(remotesInfo).reduce(
+    (prev, remoteInfo) => ({
+      total: prev.total + remoteInfo.size,
+      used: prev.used + remoteInfo.used,
+      available: prev.available + remoteInfo.available,
```

Comment:
Is this the definitive shape of the result, though? If we plan to add another field like `other`, I think it's important to add it now, even if the value is `0`, so that the expected returned object's shape is as stable as possible.